Dockerfile

```diff
@@ -1,11 +1,27 @@
-FROM python:3.8.3-slim-buster
+FROM alfg/ffmpeg
 
-ADD ./ /
+WORKDIR /app
 
-RUN apt-get -y update
-RUN apt-get -y upgrade
-RUN apt-get -y install git
-RUN apt-get -y install ffmpeg
-RUN pip install -r requirements.txt
+ENV DEBIAN_FRONTEND=noninteractive
+ENV TZ=Asia/Kolkata
```

Comment:
Maybe before this PR is merged, this line should be removed? I don't think defaulting the timezone to Asia/Kolkata is a good idea. It's probably best to leave the timezone as default or whatever the user specifies.